src/main/java/io/muserver/murp/ReverseProxyBuilder.java

```diff
@@ -92,6 +86,7 @@ public ReverseProxyBuilder sendLegacyForwardedHeaders(boolean sendLegacyForwarde
      * <p>Reverse proxies are generally supposed to forward the original <code>Host</code> header to target
      * servers, however there are cases (particularly where you are proxying to HTTPS servers) that the
      * Host needs to match the Host of the SSL certificate (in which case you may see SNI-related errors).</p>
+     *
      * @param sendHostToTarget If <code>true</code> (which is the default) the <code>Host</code> request
```

Comment:
If `true` and `ClientUtils.DISALLOWED__REQUEST_HEADERS` contains `host` then throw an exception telling the user to upgrade to Java 17 or set this to false.

Perhaps in the `build()` method just log a message noting which headers are disallowed (except for content-length)

Reply:
fixed and add log in build() method.